src/app/features/change-history/record-diff/record-diff.component.scss

```
@@ -0,0 +1,116 @@
+@use "variables/colors";
+@use "variables/sizes";
+@use "variables/breakpoints";
+
+$diff-columns: 140px minmax(0, 1fr) 28px minmax(0, 1fr);
+
+// layout (flex/align/gap) comes from utility classes in the template
+.diff-note {
+  padding: sizes.$small;
+  margin-bottom: sizes.$regular;
+  border-radius: 6px;
+  // light info-blue (design); ndb brand palette is orange
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)